pom.xml

```diff
@@ -99,18 +99,43 @@
             <groupId>org.slf4j</groupId>
             <artifactId>jul-to-slf4j</artifactId>
           </exclusion>
+          <!--Don't bundle jackson libraries into this plugin - use jackson2-api plugin (see below)-->
+          <exclusion>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jaxb-annotations</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+          </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>jackson2-api</artifactId>
+        <version>2.8.11.1</version>
+      </dependency>
+      <dependency>
+        <!--Not included in the jackson2-api plugin-->
```

Comment:
we should not do this until that is available - otherwise if parts of the dataformat get updated in the plugin then we can have incompatable versions that causes a big kaboom.